tests/testthat/test-compute_kidney.R

```diff
@@ -73,7 +73,9 @@ test_that("compute_egfr Test 7: CKD-EPI calculated on input data", {
     "P01", "P01-1001", 55, "M", "WHITE", 90.7, 96.3, "umol/L",
     "P01", "P01-1002", 52, "F", "BLACK OR AFRICAN AMERICAN", 68, 70, "umol/L",
     "P01", "P01-1003", 67, "M", "BLACK OR AFRICAN AMERICAN", 85, 77, "umol/L",
-    "P01", "P01-1004", 76, "F", "ASIAN", 60, 65, "umol/L"
+    "P01", "P01-1004", 76, "F", "ASIAN", 60, 65, "umol/L",
```

Comment:
@jeffreyad just checking - this extra line in the input tests for the new lines in the function?  Just want to make sure this scenario is covered adequately

Reply:
> @jeffreyad just checking - this extra line in the input tests for the new lines in the function? Just want to make sure this scenario is covered adequately
Yes, added a line to test for missing sex